src/ReactiveUI/Platforms/windows-common/ReactivePage.cs

```diff
@@ -100,6 +104,40 @@ class ReactivePage<TViewModel> :
                 typeof(ReactivePage<TViewModel>),
                 new PropertyMetadata(null));
 
+#if HAS_UNO
+#if ANDROID
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ReactivePage{TViewModel}"/> class.
+        /// Native constructor, do not use explicitly.
+        /// </summary>
+        /// <remarks>
+        /// Used by the Xamarin Runtime to materialize native
+        /// objects that may have been collected in the managed world.
+        /// </remarks>
+        /// <param name="javaReference">javaReference.</param>
+        /// <param name="transfer">transfer.</param>
```

Comment:
The JniHandleOwnership is mentioned here https://docs.microsoft.com/en-us/xamarin/android/platform/java-integration/working-with-jni and https://docs.microsoft.com/en-us/dotnet/api/android.runtime.jnihandleownership?view=xamarin-android-sdk-9 -- basically it indicates how the JNI system would transfer ownership of the pointer behind the scenes.

The javaReference would be a pointer to the native form instance.

Reply:
Copied from [here](https://docs.microsoft.com/en-us/dotnet/api/java.lang.object.-ctor?view=xamarin-android-sdk-9#Java_Lang_Object__ctor_System_IntPtr_Android_Runtime_JniHandleOwnership_).